pkg/sloop/server/internal/config/config.go

```diff
@@ -172,16 +173,26 @@ func (c *SloopConfig) Validate() error {
 }
 
 func loadFromFile(filename string) *SloopConfig {
-	yamlFile, err := ioutil.ReadFile(filename)
+	var config SloopConfig
+
+	configFile, err := ioutil.ReadFile(filename)
 	if err != nil {
 		panic(fmt.Sprintf("failed to read %v. %v", filename, err))
 	}
-	var config SloopConfig
-	err = yaml.Unmarshal(yamlFile, &config)
+
+	if strings.Contains(filename, ".yaml") {
```

Comment:
consider the paradigm of trying the `yaml.Unmarshal`, if it works then use it. If it fails, then try `json.Unmarshal`. If that fails, then report the error. This avoids the requirement of file extensions.